src/plugins/console/public/styles/_app.scss

```diff
@@ -41,11 +41,6 @@
   }
 }
 
-.conApp__outputPanel__emptyState {
```

Comment:
Removing this as it was added by one of the first commits from https://github.com/elastic/kibana/pull/190627 but it's not needed anymore as we use an EUI empty prompt.